src/Operator.ts

```diff
@@ -1,7 +1,11 @@
 import {Subscriber} from './Subscriber';
 
-export class Operator<T, R> {
-  call<T, R>(subscriber: Subscriber<R>): Subscriber<T> {
+export interface Operator<T, R> {
```

Comment:
@kwonoj I think this was as a result of some difference in parameter variance when implementing an interface instead of "implementing" a class. I can't recall exactly why I had this, so will remove.